tools/installers/install_whisper.sh

```diff
@@ -47,7 +47,7 @@ echo "cuda_version=${cuda_version}"
 # hash of tested commit: 9e653bd0ea0f1e9493cb4939733e9de249493cfb
 
 if "${torch_17_plus}" && "${python_38_plus}"; then
-    python -m pip install openai-whisper==20230308
+    python -m pip install openai-whisper==20250625
```

Comment:
This might conflict with https://github.com/espnet/espnet/pull/6223

Reply:
No problem, I was planning on reverting these modifications. Especially if the modifications are not directly related to the Numpy update. Additional modifications will be moved into new PRs.